packages/components/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "author": {
+    "name": "SICK Integrated Automation - UX and UI Services Team",
+    "url": "https://www.sick.com"
+  },
+  "dependencies": {
+    "@microsoft/fast-element": "^1.12.0",
+    "@microsoft/fast-foundation": "^2.49.1",
+    "rimraf": "^5.0.1",
+    "tslib": "^2.6.2"
```

Comment:
For the rimraf dependency I agree.
But if I get [this article](https://zenzes.me/today-i-learned-tslib-is-not-a-devdependency/) right, it might be useful to leave the "tslib" dependency there. Especially if the tsconfig property "importHelpers" is set to true. also see the [docu for the tsconfig](https://www.typescriptlang.org/tsconfig#importHelpers)